src/Primary_Extension/org/lobobrowser/primary/ext/TextViewerWindow.java

```diff
@@ -43,6 +49,10 @@
   private final JTextArea textArea;
   private boolean scrollsOnAppends;
 
+  // Key Stroke for CMD in case of Mac and CTRL otherwise
+  private static final int CMD_CTRL_KS = PlatformInit.OS_NAME == OS.MAC ? Toolkit.getDefaultToolkit().getMenuShortcutKeyMask()
```

Comment:
That's a good thought. Can you use `ComponentSource` to host the field then? That seems good enough for current set of changes.

If the mask gets used more widely, we can hoist it up to a larger class like `PlatformInit` later.